Source/Docs/command-line-arguments.rst

```diff
@@ -61,11 +61,13 @@ Below are the command line arguments available:
     -output-nasm,CompilerDebug.NasmFile,%DEFAULT%
     -output-asm,CompilerDebug.AsmFile,%DEFAULT%
     -output-map,CompilerDebug.MapFile,%DEFAULT%
+    -output-counters,CompilerDebug.CountersFile,%DEFAULT%
     -output-time,CompilerDebug.CompilerTimeFile,%DEFAULT%
     -output-debug,CompilerDebug.DebugFile,%DEFAULT%
     -output-inlined,CompilerDebug.InlinedFile,%DEFAULT%
     -output-hash,CompilerDebug.PreLinkHashFile,%DEFAULT%
     -output-hash,CompilerDebug.PostLinkHashFile,%DEFAULT%
+	-filter-counters,CompilerDebug.CountersFilter,{value}
```

Comment:
Option name does not match, you have `-filter-counters` here but `-counters-filter` in the C# source code

Reply:
Fixed.